src/frontends/onnx/tests/onnx_import.in.cpp

```diff
@@ -6956,5 +6956,75 @@ OPENVINO_TEST(${BACKEND_NAME}, onnx_float8e4m3fn_constant) {
     // Float8e4m3(fn) doesn't have infinity/-infinity values
     test_case.add_expected_output<ov::float8_e4m3>({-1.0f, 0.0f, 1.0f, NAN, 256.f, -256.f});
 
+    test_case.run();
+}
+
+// Basic Col2Im test
+OPENVINO_TEST(${BACKEND_NAME}, onnx_col2im) {
+    const auto model = convert_model("col2im_cpu.onnx");
+    auto test_case = ov::test::TestCase(model, s_device);
+    test_case.add_input<float>({1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f, 9.0f});
+    test_case.add_input<int64_t>({3, 3});
+    test_case.add_expected_output<float>(Shape{1, 1, 3, 3}, {1.0f, 2.0f, 3.0f, 4.0f, 5.0f, 6.0f, 7.0f, 8.0f, 9.0f});
+
+    test_case.run();
+}
```

Comment:
Each unit tests added here should have corresponding model with prototxt format (converted to .onnx automatically). Please take look at the example PRs and models tests directory: https://github.com/openvinotoolkit/openvino/tree/master/src/frontends/onnx/tests/models  

Those tests are independent from: 
src/frontends/onnx/tests/tests_python/test_backend.py